app/assets/stylesheets/application.scss

```diff
@@ -24,12 +25,27 @@ a {
   color: $primary-body;
 }
 
+.blog-text {
+  color: $blog-text;
+  margin: 0;
+}
+
 .navbar-static-top {
   position: fixed;
   z-index: 4;
   top: 0px;
 }
 
+.fixed {
+  position: fixed;
```

Comment:
Properties should be ordered height, overflow, position, right, top, width, z-index